### DISCUSSION TITLE
Virtual directories or Alias (from Filezilla) like multiple directories for a SFTP user

### DISCUSSION THREAD

**Discussion by fcastellino** [PowerShell/Win32-OpenSSH#2108] [Q&A]
Hi,

I am configuring a SFTP server that has a share for our internal domain users that upload and update pdf files. This same share has to be accessed by external consultants through SFTP. 
Is there a way to create a SFTP home folder for each external consultant with a shortcut to multiple folders in the Windows share.
For example.

Winshare folder contains Folder 1, Folder 2, Folder 3 and Folder 4

SFTP user1 is chrooted to a home folder SFTPhomeuser1 and in there I have a shortcut to Folder 1 and Folder 2
SFTP user2 is chrooted to a home folder SFTPhomeuser2 and in there I have a shortcut to Folder 3 and Folder 4
SFTP user3 is chrooted to a home folder SFTPhomeuser3 and in there I have a shortcut to Folder 1 and Folder 3
etc.

Filezilla has this functionality. However they don't have SFTP, only FTPS.

I would like to use the SFTP in Windows but how can I accomplish what I am asking. I have searched high and low on the internet but no one seems to mention one home folder with multiple shortcuts in their SFTP configuration.

Thanks for the help

**Comment by tgauth** [comment]
@fcastellino - does https://github.com/PowerShell/Win32-OpenSSH/issues/1403#issuecomment-945460625 capture your scenario?

**Reply by fcastellino** [reply]
Yes, very similar or probably the same thing, but there is no solution!! 
I did indeed create symlinks to the Winshare folder and I can browse through the files, folders, but cannot download them through SFTP, and or cannot open the files. And I gave the SFTP user read access to the share in Windows ACL.

But what is the use of SFTP in windows if the basic functionality of opening and/or downloading them is not available. Is this just because I chrooted the user? It makes no sense. Of course, we want to chroot the user. The user should not be able to go through the file structure and browse, modify files. That is the very reason we want to go from FTP to SFTP, right ?

Thanks,

**Reply by maertendMSFT** [reply]
@fcastellino what version are you using?  It sounds like you are using a network share, we fixed this for network shares in the latest release.
